tracer/build/_build/docker/gitlab/gitlab.windows.dockerfile

```diff
@@ -0,0 +1,36 @@
+﻿ARG BASE_IMAGE=mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019
```

Comment:
Uses the same base image as the existing image and installs

- VS 
- .NET FX 3.5 (Required by WIX)
- WIX
- .NET SDK